platform/feature-api/api/pod-api/.editorconfig

```diff
@@ -0,0 +1,2 @@
+[*]
+max_line_length = 100
```

Comment:
Seems like we could also put a single `.editorconfig` a bit higher up, hm? E.g. directly under _feature-api_. But also works for me like this, makes it easier to convert one package at a time to 80 columns.